pekko-connectors-sample-file-to-elasticsearch/build.sbt

```diff
@@ -0,0 +1,11 @@
+import sbt.Keys._
+
+name := "pekko-connectors-sample-file-to-elasticsearch"
+organization := "samples"
+version := "1.0.0"
+scalaVersion := Dependencies.scalaVer
+libraryDependencies ++= Dependencies.dependencies
+
+// #TODO: Remove these lines ones Pekko Connectors have 1.0.0
```

Comment:
Can you use pekko-connectors version 0.0.0+131-79ec6fa6-SNAPSHOT ? This should remove the need for the libraryDependencySchemes hack because that version is built with pekko 1.0.0.

Reply:
Resolved: [0a15746](https://github.com/apache/incubator-pekko-connectors-samples/pull/2/commits/0a15746088056c97d7b78920b9fd276cf9e31c3d)